docs/design.adoc

```diff
@@ -268,27 +268,17 @@ surplus value allows for it. For example:
 Each notifier reporting about deposit liquidation start (`notifyLiquidation`)
 or about a deposit being liquidated outside of the coverage pool
 (`notifyLiquidated`) can be rewarded with COV tokens. Those tokens represent a
-part of the asset pool ownership. The amount of token reward is set by
-the governance. It can be determined either as a fixed amount or as a
-percentage of the total COV supply. Current values of reward parameters can
-be viewed using the following methods of the risk manager:
-
+part of the asset pool ownership. Initially, the amount of token reward is zero,
+but the value can be updated by the governance (possibly multiple times).  The
```

Comment:
It will be zero for a very short time, so let's not mention it in the design doc. We may also decide to not make it zero during the deployment.

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/146/commits/98ae98c3462967491810499def1dc1bfbbff4a45